services/fromage/service_configs/fromage/environment.yml

```diff
@@ -0,0 +1,3 @@
+SERVICE_PORT: 8069
+SERVICE_NAME: fromage
+FLASK_APP: server
```

Comment:
do not forget to add here the parameter from build args which is going to be added there